nurseconnect/models.py

```diff
@@ -0,0 +1,102 @@
+from django.contrib.auth.hashers import make_password, check_password
+from django.contrib.auth.models import User
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from nurseconnect.constants import GENDERS
+from molo.commenting.models import MoloComment
+from wagtail.contrib.settings.models import BaseSetting
+from wagtail.contrib.settings.registry import register_setting
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+
+
+class NurseConnectUserProfile(models.Model):
+    user = models.OneToOneField(
```

Comment:
From the django docs. `on_delete will become a required argument in Django 2.0. In older versions it defaults to CASCADE.` Since the project currently uses v1.9.9, maybe its necessary now to add it? What do you think?